ConcernsCaseWork/ConcernsCaseWork.Service/Cases/CaseDto.cs

```diff
@@ -3,79 +3,31 @@
 namespace ConcernsCaseWork.Service.Cases
 {
 	[Serializable]
-	public sealed class CaseDto
+	public sealed record CaseDto
 	{
-		[JsonProperty("createdAt")]
-		public DateTimeOffset CreatedAt { get; }
-
-		[JsonProperty("updatedAt")]
-		public DateTimeOffset UpdatedAt { get; }
-		
-		[JsonProperty("reviewAt")]
-		public DateTimeOffset ReviewAt { get; }
-		
-		[JsonProperty("closedAt")]
-		public DateTimeOffset ClosedAt { get; }
-		
-		[JsonProperty("createdBy")]
-		public string CreatedBy { get; }
-		
-		[JsonProperty("description")]
-		public string Description { get; }
-		
-		[JsonProperty("crmEnquiry")]
-		public string CrmEnquiry { get; }
-		
-		[JsonProperty("trustUkprn")]
-		public string TrustUkPrn { get; }
-		
-		[JsonProperty("reasonAtReview")]
-		public string ReasonAtReview { get; }
-
-		[JsonProperty("deEscalation")]
-		public DateTimeOffset DeEscalation { get; }
-		
-		[JsonProperty("issue")]
-		public string Issue { get; }
-
-		[JsonProperty("currentStatus")]
-		public string CurrentStatus { get; }
-
-		[JsonProperty("caseAim")]
-		public string CaseAim { get; }
-		
-		[JsonProperty("deEscalationPoint")]
-		public string DeEscalationPoint { get; }
-		
-		[JsonProperty("nextSteps")]
-		public string NextSteps { get; }
+		public DateTimeOffset CreatedAt { get; init; }
```

Comment:
Removing the json property technically makes this a breaking change. I assume we're all good on this?
Particularly ones where the name is different by more than just case. Such as deEscalationPoint vs DeEscalation

Reply:
The case is resolved automatically. Tested and all works for me - happy for anyone else to take a look though